index.js

```diff
@@ -6,15 +7,258 @@ function ask(questionText) {
+
+  let blueprintQ = await ask('\nYou find a blueprint of the ship.\nThe blueprint says the ship\'s spare parts are in the Engine Block.\ndo you take or leave. ')
+  if (blueprintQ === 'leave') {
+    console.log('\n***You place the blueprint back down***')
+  } else {
+    console.log('\nOh no, the poison left on the blueprint spreads throughout your body\nyou have now died\nINCONCEIVABLE! ')
+    process.exit()
+  }
+
+
+  console.log((roomsLookUp[player.location].desc))
+  //---------------- Main Story code: move, take, drop, status, inventory ----------
+  while (true) {
+    let room = roomsLookUp[player.location];
+    let question = await ask('move, drink, take, drop, check inventory, read, status\nWhat do you want to do? ')
+    if (question === 'move') {
+      console.log(shipRooms[player.location].canChangeTo)
+      let clarifyingQ = await ask('\nWhat room do you want to go to? ')
+      if (clarifyingQ === 'engineBlock' && engineBlock.locked === true) {
+        await passcodeEngineBlock()
+      }
+      if (clarifyingQ === 'hangarControlRoom') {
+        turnPowerOnInHangarControlRoom()   
+      }
+      room.moveBetweenRooms(clarifyingQ)
+      room = roomsLookUp[player.location]
+      console.log(room.desc)
+    }
+    else if (question === 'take') {
+      let takeQ = await ask('\nWhat would you like to take? ')
+      if (room.inv.includes(takeQ)) {
+        console.log('\nYou have taken this item for your inventory.')
+        player.inventory.push(takeQ)
+      } else {
+        console.log('That specific item is not found in this room. Look elsewhere. ')
+      }
+    }
+    else if (question === 'drop') {
+      let dropQ = await ask('\nWhat would you like to drop? ')
+      if (player.inventory.includes(dropQ)) {
+        console.log('\n\nYou have dropped this item. ')
+        player.inventory.pop[dropQ]
+      } else {
+        console.log('\nThat specific item is not found in your inventory. ')
+      }
+    }
+    else if (question === 'check inventory') {
+      let inventoryQ = await ask('\nWould you like to check your inventory? ')
+      if (inventoryQ === 'yes') {
+        console.log('The items you have in your inventory are: ' + player.inventory)
+      } else {
+        console.log('\n\nOK ... moving on ')
+      }
+    }
```

Comment:
you could refactor this and trim it down so that when the user types 'check inventory' it would just spit the inventory out, there doesn't need to be a second check to see if they actually want to do that.

Reply:
same thing would go for your status